### PR TITLE
feat(inputs.lustre2): support reading bulk read/write stats

### LINKED ISSUE #13636: lustre: please add /proc/fs/lustre/osd-*/*/brw_stats
### Use Case

In the past, with other monitoring solutions, I have been able to monitor the "brw"/"bulk read/write"/"bulk IO" stats that they provide in `brw_stats` files. I can't currently do that with the current plugin because these files aren't monitored.

### Expected behavior

I want to be able to generate graphs like this:
image

NOTE: this was actually done with a patched version of telegraf that I'm running locally. Once I get the CLA sorted out, I plan to file an MR to solve this issue.

### Actual behavior

None of that data is available.

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by DStrand1**
## Summary

Continuation of #7733

## Checklist

- [X] No AI generated code was used in this PR

## Related issues

resolves #13636

### CODE REVIEW

File: plugins/inputs/lustre2/README.md [hunk 1]
```diff
@@ -22,18 +22,22 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
 # This plugin ONLY supports Linux
 [[inputs.lustre2]]
   ## An array of /proc globs to search for Lustre stats
-  ## If not specified, the default will work on Lustre 2.5.x
+  ## If not specified, the default will work on Lustre 2.12.x
```
Comment (powersj) [inline plugins/inputs/lustre2/README.md:25]: Does this mean this won't work on versions from 2.5 to 2.12?

File: plugins/inputs/lustre2/lustre2.go [hunk 2]
```diff
@@ -440,6 +464,98 @@ func (l *Lustre2) GetLustreProcStats(fileglob string, wantedFields []*mapping) e
...
+			// snapshot_time should be discarded
+			if strings.Contains(parts[0], "snapshot_time") {
+				continue
+			}
+
+			// This is a bucket for a given header
+			for _, wanted := range wantedFields {
+				if headerName != wanted.inProc {
+					continue
+				}
```
Comment (srebhan) [inline plugins/inputs/lustre2/lustre2.go:517]: Wouldn't it be easier to make `wantedFields` a map and look-up the `headerName`?

**Review by srebhan — COMMENTED** [review]
@DStrand1 thanks for the code, just one small question from my side...

File: plugins/inputs/lustre2/lustre2.go (hunk 2 above)
Comment (DStrand1) [reply plugins/inputs/lustre2/lustre2.go:517]: I agree, I don't like the way this code is laid out currently with the field mappings and I plan to refactor it in a follow-up PR. For now I just followed how the other mappings were done, but I can refactor this here instead if you'd prefer

**Review by srebhan — APPROVED** [review]
Thanks @DStrand1!

**Pull request merged**